pkg/inject/oauth-proxy.go

```diff
@@ -44,6 +44,19 @@ func getOAuthProxyContainer(jaeger *v1.Jaeger) corev1.Container {
 		"--upstream=http://localhost:16686",
 	}
 
+	volumeMounts := []corev1.VolumeMount{{
+		MountPath: "/etc/tls/private",
+		Name:      service.GetTLSSecretNameForQueryService(jaeger),
+	}}
+
+	if len(jaeger.Spec.Ingress.OpenShift.HtpasswdFile) > 0 {
+		args = append(args, fmt.Sprintf("--htpasswd-file=%s", jaeger.Spec.Ingress.OpenShift.HtpasswdFile))
+		args = append(args, "--display-htpasswd-form=false")
```

Comment:
Just to confirm - this doesn't affect the Jaeger UI login?

Reply:
No, users would still need to click the "Login with OpenShift" button and will be directed to the login page hosted at OpenShift, which will redirect the users back to the Jaeger UI.

Reply:
Ok great.